doc/user/zebra.rst

```diff
@@ -351,6 +360,143 @@ default) should the specified gateways not be reachable. Eg:
 
 .. _multicast-rib-commands:
 
+VRF Commands
+============
+
+Currently, the user has the possibility to configure VRFs. VRF is a way to
+separate networking contexts on the same machine. Those networking contexts
+stick to separate interfaces, thus making it possible to associate one interface
+to a specific VRF.
+VRF can be used, for example, when instantiating per enterprise networking
+services, without having to instantiate the physical host machine or the
+routing management daemons for each enterprise. As a result, interfaces are
+separate for each set of VRF, and routing daemons can have their own context
+for each VRF.
+
+This conceptual view introduces the `Default VRF` case. If the user does not
+configure any specific VRF, then by default, the user will however configure
+the `Default VRF`.
+On the Zebra context, this can be done when being in configuration mode, when
+configuring a static route `ip route NETWORK GATEWAY`.
+
+   .. code-block:: frr
+
+      # case without VRF
+      configure terminal
+      ip route 10.0.0.0 255.255.255.0 10.0.0.2
+      exit
+
+Configuring VRF networking contexts can be done on various ways on FRR.
+The VRF interfaces can be configured by entering in interface configuration
+mode : `interface INTERFACE VRF`. Also, if the user wants to configure a static
```

Comment:
I don't believe such a command exists, the real one appears to be `interface IFNAME vrf NAME`. However, it's not documented yet (http://frrouting.readthedocs.io/en/latest/zebra.html#clicmd-interfaceIFNAME). If you update the doc on that command, then you can cross reference it like so:
```
:clicmd:`interface IFNAME vrf NAME`
```

Reply:
does that mean adding following ion zebra.rst ?
```
.. index:: interface IFNAME vrf	VRF

.. clicmd:: interface IFNAME vrf VRF
```